js/models/messages.js

```diff
@@ -655,11 +655,13 @@
           !this.get('isPublic') ||
           this.getConversation().isModerator(this.OUR_NUMBER) ||
           this.getSource() === this.OUR_NUMBER,
+        hasModPerms: this.getConversation().isModerator(this.OUR_NUMBER),
```

Comment:
We can take advantage of the cached `const conversation = this.getConversation();` on line 615.
Also, `this.getConversation().isModerator(this.OUR_NUMBER)` seems to be used by `isDeletable` above so that would warrant caching it as well?